### PR TITLE
 fix: ui changes for RLS policy modal

### PR DESCRIPTION
**Pull request by likendev**
## What kind of change does this PR introduce?

* added cancel button
* moved template button to left
* added close button

## What is the new behavior?

image

## Additional context

Issue #18304

### CODE REVIEW

File: studio/components/interfaces/Auth/Policies/PolicySelection.tsx [hunk 1]
```diff
@@ -58,18 +60,23 @@ const PolicySelection = ({
...
+              rel="noreferrer"
+              className="text-brand transition-colors hover:text-brand-600"
+              href="https://supabase.com/docs/guides/auth#policies"
+            >
+              here
+            </a>
+            .
+          </p>
+          <div className="flex justify-end w-full items-center space-x-3" onClick={onSelectCancel}>
+            <Button type="default">Cancel</Button>
```
Comment (MildTomato) [inline studio/components/interfaces/Auth/Policies/PolicySelection.tsx:77]: move onClick to `<Button>`

**Review by MildTomato — DISMISSED** [review]
Hey @likendev - just 1 change maybe required.

File: studio/components/interfaces/Auth/Policies/PolicySelection.tsx (hunk 1 above)
Comment (likendev) [reply studio/components/interfaces/Auth/Policies/PolicySelection.tsx]: Hi, thanks for reviewing the code, overlooked on this. Have amended the code!

**Review by likendev — COMMENTED** [review]
All good. Thanks!

**Review by likendev — COMMENTED** [review]
Thanks for the changes

**Comment by alaister** [comment]
Hey @likendev,

Thanks again for the PR, and sorry for the delay!

We're currently working on replacing this UI entirely, and this has gotten quite stale, so I'm going to go ahead and close this PR.

Please don't hesitate to open up a new PR if the changes are still relevant.

**Pull request closed without merging**